pkg/utils/common.go

```diff
@@ -41,6 +41,7 @@ const (
 	NamespaceNameFormat    = fleetPrefix + "member-%s"
 	RoleNameFormat         = fleetPrefix + "role-%s"
 	RoleBindingNameFormat  = fleetPrefix + "rolebinding-%s"
+	ValidationPathFmt      = "/validate-%s-%s-%s"
```

Comment:
Suggest to move this def to the pkg/webhook/validation, "validation" package and create a new file named, common.go.

Please add a comment for the public variable/const/func.